wire-schema/src/main/java/com/squareup/wire/schema/SchemaLoader.java

```diff
@@ -156,9 +160,29 @@ private Schema loadFromDirectories(Map<Path, Path> directories) throws IOExcepti
       }
     }
 
+    ProtoFile descriptor = loaded.get(DESCRIPTOR_PROTO);
+    if (descriptor == null) {
+      loaded.put(DESCRIPTOR_PROTO, loadDescriptorProto());
+    }
+
     return new Linker(loaded.values()).link();
   }
 
+  /**
+   * Returns Google's protobuf descriptor, which defines standard options like default, deprecated,
+   * and java_package. If the user has provided their own version of the descriptor proto, that is
+   * preferred.
```

Comment:
Unfortunately most calling code won't have it — and shouldn't have it. It's only necessary if you're defining custom extensions on things.